UK TL: More new options

tsunamistate opened the pull request:
@SourceCodeDeleted please take a look

AJenbo commented:
68%, it's getting close to the magic 75% required to be listed :)

@SourceCodeDeleted do you have time to review the update?

SourceCodeDeleted commented on Translations/uk.po:
```diff
@@ -1200,145 +1208,142 @@ msgstr "Допомога недоступна"
-msgstr "Швидке Послання {}"
+#: Source/diablo.cpp:1423
+msgid "Speedbook"
+msgstr "Швидка Книга Магії"
```
The rest of the translateion uses
Чар
Can we continue to use this word?

tsunamistate replied:
Fixed

SourceCodeDeleted commented on Translations/uk.po:
```diff
@@ -1200,145 +1208,142 @@ msgstr "Допомога недоступна"
-msgstr "Інформація про гру"
+#: Source/diablo.cpp:1487
+msgid "Quick Message {}"
+msgstr "Швидке Послання {}"
```
Do you think повідомлення would be better?

tsunamistate replied:
Too long

tsunamistate replied:
Done

SourceCodeDeleted commented on Translations/uk.po:
```diff
@@ -1200,145 +1208,142 @@ msgstr "Допомога недоступна"
-msgstr "Показує інформацію про гру."
+#: Source/diablo.cpp:1488
+msgid "Use Quick Message in chat."
+msgstr "Відправити Швидке Послання в чат."
```
повідомлення perhaps?

tsunamistate replied:
I thought it was too long, but here's the screenshot with your suggestion
![image](https://user-images.githubusercontent.com/4958520/148641605-1d775699-f86b-4c47-b693-b2a91abec2f5.png)

Looks OK, so I have changed it to your suggestion

SourceCodeDeleted commented:
Ok most of this looks good. @tsunamistate I made a small suggestion, how do you think?>

tsunamistate commented:
Fixed inconsistencies pointed out. Turns out I left one TL incomplete, so I fixed that too.
@SourceCodeDeleted as for your suggestion, I thought it would be too long in UI, but play testing proved me wrong, so I changed it.
@AJenbo fixed and rebased, good to merge
